docs/automated-process.md

```diff
@@ -149,3 +149,7 @@ next to the release version:
 
 2. Set `existing-branch` to the branch you pushed to the `ansible-build-data`
    repository.
+
+Finally, remember to prepare the directory for the next major Ansible release. How
+to do this is described in [Setting up for a new major
+release](new-ansible-and-freezes.md#setting-up-for-a-new-major-release).
```

Comment:
I'm not 100% sure about the "finally" here. When should this be done? When the new major release is available on PyPI? At least, that's how I always understood this.

Maybe we should make this clearer by saying "When the new major release has been done and is available on PyPI, remember to prepare..." or something similar.

Reply:
Basically it can be done once the PR has been merged, i.e. as soon as the `X/ansible-X.0.0.deps` file is there. I'll update the formulation later.

Reply:
Updated in af5d0338e652235e72ce3e6ad53befce531aa19f.